Vulkan-Projects.adoc

```diff
@@ -0,0 +1,162 @@
+
+* Repository: https://github.com/KhronosGroup/MoltenVK
+* License: [Apache](https://github.com/KhronosGroup/MoltenVK/blob/master/LICENSE)
+* Contacts: bill.hollings@brenwill.com
+* Compiler requirements: Xcode 9 / python 3
+
+## Khronos/Vulkan-HPP
+Vulkan-Hpp is a set of lightweight C++ bindings for the Vulkan API.
+
+* Repository: https://github.com/KhronosGroup/Vulkan-Hpp
+* License: [Apache](https://github.com/KhronosGroup/Vulkan-Hpp/blob/master/LICENSE.txt)
+* Contact: Markus Tavenrath <mtavenrath@nvidia.com>
+* Compiler requirements: MSVC 2013+ / gcc 4.8.2+ / clang 3.3+
+
+## Khronos/SPIRV-Cross
+SPIRV-Cross is a practical tool and library for performing reflection on SPIR-V and
+disassembling SPIR-V back to high level languages.
+
+* Repository: https://github.com/KhronosGroup/SPIRV-Cross
+* License: [Apache](https://github.com/KhronosGroup/SPIRV-Cross/blob/master/LICENSE)
+* Contacts:
+	* hans-kristian.arntzen@arm.com (@HansKristian-ARM)
+* Compiler requirements: MSVC 2013 / gcc 4.8/4.9+ / clang 3.x+
+
+## DirectX Shader Compiler (DXC)
+DirectX Shader Compiler (DXC) is Microsoft's next-gen official HLSL compiler, based on LLVM/Clang.
+Apart from compiling HLSL into DXIL, it can also compile HLSL into SPIR-V, thanks to contribution from Google.
+
+* Landing page: https://github.com/Microsoft/DirectXShaderCompiler/wiki
+* Repository: https://github.com/Microsoft/DirectXShaderCompiler
+* License: University of Illinois Open Source License
+* Contacts:
+  * Lei Zhang <antiagainst@google.com> (for SPIR-V CodeGen)
+  * opencode@microsoft.com (for other issues)
+* Platform: Windows, Linux, macOS
+* Compiler requirements: MSVC 2017 / GCC 5.5+ / Clang 3.8+
+* Download: https://khr.io/dxcappveyorbuild (rolling release for Windows)
+
+## RenderDoc
+RenderDoc - a graphics debugger, currently available for Vulkan, D3D11, D3D12, and OpenGL development on Windows 7 - 10 and Linux.
+
+* Website: https://renderdoc.org/
+* Repository: https://github.com/baldurk/renderdoc
+* License: [MIT](https://github.com/baldurk/renderdoc/blob/v0.x/LICENSE.md)
+* Contacts:
+    * baldurk@baldurk.org
+    * [#renderdoc on freenode IRC](https://kiwiirc.com/client/irc.freenode.net/#renderdoc)
+* Code of conduct: [contributor covenant](https://github.com/baldurk/renderdoc/blob/v0.x/CODE_OF_CONDUCT.md)
+* Compiler requirements: MSVC 2015 / gcc 5 / clang 3.4
+
+## LunarG/VulkanTools
+Source code for various Vulkan Tools: vktrace/vkreplay, device simulation layer, API dump layer, fps monitor layer, screenshot layer, assistant layer, layer factory framework, and Vulkan installation analyzer.
+
+* Repository: https://github.com/LunarG/VulkanTools
+* License: [Apache](https://github.com/LunarG/VulkanTools/blob/master/LICENSE.txt)
+* Contact: David Pinedo <david@lunarg.com>
+* Code of conduct: https://github.com/LunarG/VulkanTools/blob/master/GOVERNANCE.md
+* Compiler requirements: MSVC 2013+ / gcc 4.8.2+ / clang 3.4+
+
+## gfx-rs
```

Comment:
ISTM good to segregate the official Khronos repositories from related stuff like this - shouldn't this be on the Vulkan website under developer resources? The issue with external projects is that it's so open-ended, and hard to keep track of what's popular, what's maintained, etc.

Reply:
I deleted gfx-rs for from this doc - it is listed on the Vulkan website already